packages/bs-flopflip/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "private": true,
+  "name": "bs-flopflip",
+  "version": "0.1.0",
```

Comment:
Just as a note: I'd prolly release this as 1.0.0 straight away. Not a big fan of 0.x.x things due to different matching with ranges.

Reply:
Fine for me. We can also mark it as "alpha" or something if you want. I don't mind 😇